src/Cortside.Common.Messages/Models/ErrorsModel.cs

```diff
@@ -15,10 +13,8 @@ public class ErrorsModel {
             Errors = new List<ErrorModel>();
         }
 
-        public ErrorsModel(ModelStateDictionary modelState) {
-            Errors = modelState.Keys
-                        .SelectMany(key => modelState[key].Errors.Select(x => new ErrorModel(x.Exception?.GetType()?.Name ?? "ModelStateValidation", key, x.ErrorMessage)))
-                        .ToList();
+        public ErrorsModel(List<ErrorModel> errors) {
```

Comment:
This behavior was moved to the AspNetCore project:

`    public class ModelStateResult : ObjectResult {
        public ModelStateResult(ModelStateDictionary modelState) : base(new ErrorsModel(modelState.Keys
                        .SelectMany(key => modelState[key].Errors.Select(x => new ErrorModel(x.Exception?.GetType()?.Name ?? "ModelStateValidation", key, x.ErrorMessage)))
                        .ToList())) { }
    }`